src/lumigo_tracer/auto_instrument_handler.py

```diff
@@ -6,8 +6,19 @@
     # Try to import AWS's current _get_handler logic
     from bootstrap import _get_handler as aws_get_handler
 except Exception:
-    # Import a snapshot of _get_handler from python38 runtime
-    from lumigo_tracer.libs.bootstrap import _get_handler as aws_get_handler
+    # Only import the logger if we need it, no need to slow down the import time
+    from lumigo_core.logger import get_logger
```

Comment:
I only import the logger if we need to log a warning because I don't want to complicate this basic file with extra imports & logic if we don't need to

Reply:
there is `warn_client`, which I think is what you want